app/screens/people-screen/contacts/contacts-card.tsx

```diff
@@ -0,0 +1,174 @@
+            username: contact.username,
+          })
+        }
+        name="send"
+        size="medium"
+        iconOnly
+      />
+    </View>
+  )
+}
+
+export const ContactsCard = () => {
+  const styles = useStyles()
+
+  const { LL } = useI18nContext()
+
+  const isAuthed = useIsAuthed()
+  const navigation = useNavigation<StackNavigationProp<PeopleStackParamList>>()
+
+  const { loading, data, error } = useContactsCardQuery({
+    skip: !isAuthed,
+    fetchPolicy: "cache-and-network",
+  })
+
+  if (error) {
+    toastShow({ message: error.message })
+  }
+
+  const contacts = useMemo(() => (data ? getRecentContacts(data) : []), [data])
+
+  return (
+    <View style={styles.container}>
+      <View>
+        <View style={styles.contacts}>
+          <Text type="h2">{LL.PeopleScreen.recentContacts()}</Text>
+        </View>
+        <View style={[styles.separator, styles.spaceTop]}></View>
+      </View>
+      {loading ? (
+        <ActivityIndicator />
+      ) : contacts.length === 0 ? (
+        <Text type="p2">{LL.PeopleScreen.noContactsTitle()}</Text>
+      ) : (
+        <>
+          <View style={styles.contactsOuterContainer}>
+            {contacts.map((contact) => (
+              <Contact key={contact.id} contact={contact as UserContact} />
+            ))}
+          </View>
+          <GaloySecondaryButton
+            title={LL.PeopleScreen.viewAllContacts()}
+            onPress={() => navigation.navigate("allContacts")}
+          />
+        </>
+      )}
+    </View>
+  )
+}
+
+const useStyles = makeStyles(({ colors }) => ({
+  container: {
+    backgroundColor: colors.grey5,
+    display: "flex",
+    flexDirection: "column",
+    marginBottom: 20,
+    borderRadius: 12,
+    padding: 12,
+    justifyContent: "center",
+    rowGap: 14,
+  },
+  contacts: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  separator: {
+    height: 1,
+    backgroundColor: colors.grey4,
+  },
+  spaceTop: {
+    marginTop: 8,
+  },
+  textCenter: {
+    textAlign: "center",
+  },
+  contactsOuterContainer: {
+    display: "flex",
+    flexDirection: "column",
+    rowGap: 10,
+  },
+  contactContainer: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    paddingTop: 3,
+  },
+}))
+
+// ---- HELPERS ----
+const getRecentContacts = (data: ContactsCardQuery) => {
+  // Extract the contacts
+  const _contacts = data?.me?.contacts || []
+  const contacts = [..._contacts] // Convert from readyonlyarray to regular array
+
+  // Sort contacts by the `createdAt` timestamp in descending order
+  contacts.sort((a, b) => {
+    if (a.transactions?.edges && b.transactions?.edges) {
+      const aDate = a?.transactions?.edges[0]?.node?.createdAt || 0
+      const bDate = b?.transactions?.edges[0]?.node?.createdAt || 0
+      return bDate - aDate
```

Comment:
previously we use to put the one where most transaction were occurring first. I think that is better than createdAt, because some older user may be the one you will interact with the most.

Reply:
basically using `transactionsCount` property

Reply:
Then we should rename from "Recent Contacts" to "Frequently Used Contacts", will bring up at all hands.